clean_project/clean_file.py

```diff
@@ -0,0 +1,36 @@
+import argparse
+import codecs
+import re
+
+
+def clean_up(src_path, dest_path):
+    """
+    This method is created to open a file, clean up and write it in another new file
+    :param src_path: source path
+    :param dest_path: destination path of new file
+    """
+    with codecs.open(src_path, "r", "utf-8") as source_file:
```

Comment:
Super nit: you should use single quotes for strings everywhere and reserve double quotes for docstrings or string blobs